### PR TITLE
PI-377 Updated to publish to the HMPPS Domain Events topic

### PR DESCRIPTION
**Pull request by marcus-bcl**

### CODE REVIEW

File: helm_deploy/offender-case-notes/values.yaml [hunk 1]
```diff
@@ -43,10 +43,10 @@ generic-service:
       DATABASE_PASSWORD: "database_password"
       DATABASE_NAME: "database_name"
       DATABASE_ENDPOINT: "rds_instance_endpoint"
-    offender-events-topic:
-      HMPPS_SQS_TOPICS_OFFENDEREVENTS_ACCESS_KEY_ID: "access_key_id"
-      HMPPS_SQS_TOPICS_OFFENDEREVENTS_SECRET_ACCESS_KEY: "secret_access_key"
-      HMPPS_SQS_TOPICS_OFFENDEREVENTS_ARN: "topic_arn"
+    domain-events-topic:
+      HMPPS_SQS_TOPICS_DOMAIN_EVENTS_ACCESS_KEY_ID: "access_key_id"
```
Comment (mikehalmamoj) [inline helm_deploy/offender-case-notes/values.yaml:47]: I think you might need to drop the underscore in `DOMAIN_EVENTS`. Spring converts these env vars into properties (in this case the matching property is `domain-events` which also neeeds to change). But we've found that sometimes Spring gets tripped up by separators so to be safe we don't use them on queue and topic names.

File: src/test/resources/application-test.yml [hunk 2]
```diff
@@ -56,3 +58,5 @@ hmpps.sqs:
   topics:
     offenderevents:
       arn: arn:aws:sns:eu-west-2:000000000000:${random.uuid}
+    domain-events:
```
Comment (mikehalmamoj) [inline src/test/resources/application-test.yml:61]: As above, I think this needs to be `domainevents`.

File: helm_deploy/offender-case-notes/values.yaml (hunk 1 above)
Comment (marcus-bcl) [reply helm_deploy/offender-case-notes/values.yaml]: Thanks, good call - I've updated those.

I believe this is to do with k8s not handling environment variables with hyphens.  See https://github.com/kubernetes/kubernetes/issues/16863

**Comment by marcus-bcl** [comment]
Converting to draft to prevent it from being merged until the comments in Slack are addressed (https://mojdt.slack.com/archives/C02S71KUBED/p1667290934919539).

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**